appium/common/helper.py

```diff
@@ -39,13 +41,14 @@ def extract_const_attributes(cls):
     """
     Return dict with constants attributes and values in the class (e.g. {'VAL1': 1, 'VAL2': 2})
```

Comment:
Is it better to address the order here?
_Return ordered dict with ..._

Reply:
At least IMO, it's unnecessary for users since not guaranteed order is due to dict behavior. (,,,but might update in next PR.)